tests/integration/test_interaction.py

```diff
@@ -695,6 +699,27 @@ def test_per_item_feedback_multiple_misplaced(self):
         for message_element in expected_message_elements:
             self.assertIn(message_element, self._get_popup_content().get_attribute('innerHTML'))
 
+    def test_submit_disabled_during_drop_item(self):
+        def delayed_drop_item(item_attempt, suffix=''):  # pylint: disable=unused-argument
+            # some delay to allow selenium check submit button disabled status while "drop_item"
+            # XHR is still executing
+            time.sleep(0.1)
```

Comment:
@cahrens I don't want to wait for something to happen - instead, the idea here is to perform checks while XHR is still in progress; without this slowdown, XHR completes too fast, so the test is extremely unreliable and have ~50% success ratio.